tests/acceptance/features/apiSharingNg/removeAccessToDriveItem.feature

```diff
@@ -18,7 +18,7 @@ Feature: Remove access to a drive item
       | sharee          | Brian              |
       | shareType       | user               |
       | permissionsRole | <permissions-role> |
-    When user "Alice" removes the share permission of user "Brian" from resource "<resource>" of space "Personal" using the Graph API
+    When user "Alice" removes the access of user "Brian" from resource "<resource>" of space "Personal" using permissions endpoint of the Graph API
```

Comment:
for resources, let's use the general step name (because we don't have multiple endpoints for that)
```suggestion
    When user "Alice" removes the access of user "Brian" from resource "<resource>" of space "Personal" using the Graph API
```